### ISSUE TITLE
VALIDATOR  FAILS

### ISSUE THREAD

**Issue by SLiMDataWrangler** [validator/validator#2026]
### Summary

The HTML validator doesn't find any errors, even when I deliberately include errors with the upload.
  
Is it functioning correctly?  It always returns no errors found whenever I access it.

Pick any page on my webiste:  https://www.okthepk.ca/home.htm

### URL for document that reproduces the problem

https://www.okthepk.ca/home.htm

### HTML/CSS/SVG snippet that reproduces the problem

```html
ANY CODE
```

### Checker error messages

```shell
It never produces an error !
```

### Steps to reproduce

1.  Create you own HTML page with an error then check it.
2.  Did it find the error?
3.  It finds no errors in my code no matter what.

### Expected behavior

I expect the validator to find errors.

### Actual behavior

It doesn't find ANY errors.

**Comment by j9t** [comment]
@SLiMDataWrangler, had only a quick look just now. The code seem valid indeed, yet when I change something (like a bogus element name), the validator does throw an error.

Can you point us to a URL where you included an obvious error and the validator doesn’t catch it? That would be faster to check than taking samples and manually auditing.

**Comment by j9t** [comment]
Just in case, the validator does not check on full appropriate use of all elements and attributes. So if this is what you’d expect to get an error for (say, using a paragraph element to mark up a heading), the validator would not report this.

**Comment by SLiMDataWrangler** [comment]
OK...  will do...  give me a while to get that done please...  SLiM


***@***.***

[signatureImage]

Canadian Railway News
www.okthepk.ca<http://www.okthepk.ca/>

Sooke News
www.okthepk.ca/sooke/news.htm<http://www.okthepk.ca/sooke/news.htm>

________________________________

**Comment by SLiMDataWrangler** [comment]
I've created this web page and inserted the code below and attached the 5 required images plus the web page.

(test01.jpg - test02.gif to test05.gif and testValidator.htm)

When you validate the page the W3C validator shows no errors.

![Image](https://github.com/user-attachments/assets/8a9d53dd-04c8-46dd-b855-2c2734ae0c38)
![Image](https://github.com/user-attachments/assets/00d328ba-cea0-4cc2-a7ef-14e5ce853a8f)
![Image](https://github.com/user-attachments/assets/ecda8c07-074f-47a8-bca9-65d1b70a8927)
![Image](https://github.com/user-attachments/assets/f0d9ea2e-d6ef-4348-815a-5c4a19c82196)
![Image](https://github.com/user-attachments/assets/73401dc0-8d00-4fee-b622-f4b9c3617148)
[testValidator.htm](https://github.com/user-attachments/files/24557100/testValidator.htm)

```html
<!DOCTYPE html>

<!-- testValidator.htm -->

<html lang="en">
  <head>
    <title> OKthePK - W3C HTML Validator Test </title>
    <meta name="description" content="An HTML page containing deliberate errors to test the W3C validator.">
    <meta name="data wrangler" content="William Slim">
    <meta name="created" content="11 January 2026">
    <meta name="revised" content="11 January 2026">
    <meta name="keywords" content="hyper,text,markup,language,html,code,w3c,validator,">
    <meta name="audience" content="Website authors, developers, coders, script kiddies, etc.">
    <meta name="robots" content="index,follow,noarchive">
    <meta charset="UTF-8">
    <meta name="msapplication-confg" content="/browserconfg.xml">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="icon" type="image/png" sizes="48x48" href=["http://www.okthepk.ca/images/iconOkPk.png"](http://www.okthepk.ca/images/iconOkPk.png)>
    <style>
      /* newsPop.css - 11 January 2026 Copyright William Slim */
      * {box-sizing:border-box;}
      body {margin:0px; font-family:Arial,Helvetica,sans-serif; font-size:20px; color:#000000;
        background-color:#999999;}
      .page {width:100%; margin:auto; background-color:#eeeeee; border:1px solid black; max-width:980px;}
      .content {width:99%; margin:auto; margin-top:30px; padding-left:30px; padding-right:30px;}                                                                
      .source {width:360px; margin-top:30px; margin-left:0px;}
      .sourceFails {margin-left:0px; font-family:"Arial Black",Gadget,sans-serif; font-size:30px;
        font-weight:bold; text-align:left; color:#ff0000; text-shadow:2px 2px #000000;}
      .sourceCaption {margin-left:0px; font-family:"Arial Black",Gadget,sans-serif; font-size:16px;
        text-align:left; color:#000000;}
      .headline {width:100%; margin-top:10px; margin-left:0px; font-family:"Arial Black",Gadget,sans-serif;
        font-size:30px; font-weight:bold; color:#000000; text-align:left;}
      .date {width:250px; margin-top:10px; margin-left:0px; font-family:"Arial Black",Gadget,sans-serif;
        font-size:18px; color:#000000; font-weight:bold; text-align:left;}
      .article {width:100%; margin-top:0px; margin-left:0px; text-align:left;}

      /* containers */
      .containerJavaScript {width:296px; margin:auto; margin-bottom:20px; padding:10px;
        border:2px solid #000000; border-radius:10px; color:#ff0000; background-color:#ffffff;
        font-family:"Arial Black",Gadget,sans-serif; font-size:20px; font-weight:bold; color:#ff0000;
        text-align:center;}
      .containerFoto {width:100%; margin:auto; text-align:center; vertical-align:top;}
      .containerNoteOne {width:100%; margin-top:30px; margin-left:0px; font-family:Verdana,Geneva,sans-serif;
        font-size:12px; text-align:left;}
      .containerNoteTwo {width:100%; margin-top:7px; margin-left:0px; font-family:Verdana,Geneva,sans-serif;
        font-size:12px; text-align:left;}
      .copyrightAct {width:100%; margin-top:20px; margin-left:0px; font-family:Verdana,Geneva,sans-serif;
        font-size:12px; text-align:center;}
      .containerAd {width:30%; margin-top:5px; margin-left:30px; margin-bottom:10px; float:right;
        text-align:left; max-width:200px;}
      .containerInsert {width:160px; margin-top:5px; margin-left:30px; margin-bottom:10px; padding:15px;
        float:right; text-align:left; border:1px solid black; background-color:#3366cc;
        font-family:Verdana,Geneva,sans-serif; color:#ffffff; font-size:13px; border-radius:6px;}
      .footer {width:100%; margin:auto; margin-top:25px; margin-bottom:140px; text-align:center;
        max-width:180px;}  

      /* responsive text - if screen width is 1370px wide or more set the font-size 22px */
      @media screen and (min-width:1370px) {div.responsiveText {font-size:22px; text-align:left;}}
      /* responsive text - If screen width is 1370px wide or less set the font-size to 18px */
      @media screen and (max-width:1370px) {div.responsiveText {font-size:18px; text-align:left;}}
  
      /* buttons */  
      div.butNav a {margin-right:25px; font-family:"Arial Black",Gadget,sans-serif; font-size:18px;
        font-weight:bold; color:#ffffff; background-color:#3366cc; border-radius:4px; text-decoration:none;
        vertical-align:bottom; padding-top:5px; padding-bottom:6px; border:1px solid black;
        border-bottom:2px solid black; border-right:2px solid black;}
      div.butNav a:hover {color:#3366cc; background-color:#ffffff;}

      /* popup exit */
      .exitPopup {position:relative; display:inline-block;}
      .exitPopup .exitPopupImg {visibility:hidden; width:165px; background-color:transparent;
        text-align:center; position:absolute; z-index:1; left:0%; top:0%;}
      .exitPopup:hover .exitPopupImg {visibility:visible; border-radius:4px; border:1px solid black;
        border-bottom:2px solid black; border-right:2px solid black;} 

      /* popup footer */
      .footerPopup {position:relative; display:inline-block; border-radius:4px;}
      .footerPopup .footerPopupImg {visibility:hidden; width:165px; background-color:#ff0000;
        text-align:center; position:absolute; z-index:1; left:-30%; top:-20px;}
      .footerPopup:hover .footerPopupImg {visibility:visible;}

      /* images border */
      .pic922border img {width:100%; height:auto; border:1px solid black; vertical-align:top; max-width:922px;}
  
      /* images NO border */
      img.noBorder {border-style:none;}
    
      /* others */
      .pageBreak {clear:both;}
      .centre {text-align:center;}
      b.fontRed {color:#ff0000; text-shadow:1px 1px #000000;}

      /* anchor colours */
      a {text-decoration:none;}
      a:link {color:#3366cc; text-shadow:1px 0px #000000;}
      a:active {color:#3366cc; text-shadow:1px 0px #000000;}  
      a:visited {color:#6699cc; text-shadow:1px 0px #000000;}
      a.sourceLink {font-family:"Arial Black",Gadget,sans-serif; font-size:30px; font-weight:bold;
        text-align:left;color:#ff0000; text-shadow:2px 2px #000000;}
    </style>
  </head>
  <body><br>
    <div class="page">
      <div class="content">
        <!-- javascript warning -->
        <noscript>
          <div class="containerJavaScript">Enable your browser's<br>
          JavaScript to display<br>
          website correctly.</div>
        </noscript>
        <div class="responsiveText"><!-- Responsive Text Start -->
<!-- PHOTO AND CAPTION CONTAINER -->
          <div class="containerFoto">
            <div class="pic922border"><img src="test01.jpg" width="920" height="460" alt="" title="">
            </div>
          </div>
          <div class="source">
<!-- SOURCE -->
            <a href=["https://www.okthepk.ca/index.htm"](https://www.okthepk.ca/index.htm) target="_blank"><b class="exitPopup"><img 
            class="noBorder" src="test02.gif" width="220" height="67" alt=""
            title=""><img class="exitPopupImg" src="test03.gif" alt=""></b></a>
          </div>
          <div class="headline">
<!-- HEADLINE -->
            <b>THIS IS THE NEWS HEADLINE</b>
          </div>
          <div class="newsDateSpace"></div>
          <div class="date">
<!-- DATE -->
            11 January 2026
          </div>
          <div class="article">
<!-- ARTICLE TEXT -->
<p><b class="fontRed">This HTML page, containing deliberate errors shown below in red, validated by file 
upload from the Microsoft Edge extension link which appears on the 
toobar returns no errors, only Info messages.<br>&nbsp;<br>

Further, the validation info displays trailing slashes when there are no trailing slashes in the original
code. So where did the slashes come from? Microsoft Edge? Unknown.</b></p>

<p><b>*&nbsp; *&nbsp; *&nbsp; *&nbsp; *</b></p>

<p><b class="fontRed">The p is missing from the &#x3C;p&#x3E; tag in the sentence below.</b></p>

            <><b>Canada</b> - This fake Canadian railway news article web page was created for testing the 
W3C HTML validator.<br>&nbsp;<br>

Once upon a time the quick fox jumped over the lazy doggy.<br>&nbsp;<br>

Once upon a time the quick fox jumped over the lazy doggy.<br>&nbsp;<br>

<b>Author unknown</b>.</p>
          </div>

<p><b class="fontRed">The &#x3C;a href=" "&#x3E; tag is blank in the &quot;&#8679; to top&quot; button 
below.</b></p>
          <!-- navigation buttons -->
          <div class="butNav">
            <a href=" "><b class="fontSubTitle">&nbsp; &#8679; to Top &nbsp;</b></a>
          </div>
<!-- NOTE ONE -->
          <div class="containerNoteOne">
            <b>&#42;</b>1. Suitable news image inserted.<br>(likely no image with original article)
          </div>
<!-- NOTE TWO -->
          <div class="containerNoteTwo">
            <b>&#42;</b>2. Original news image replaced.<br>(usually because it's been seen before)
          </div>
<!-- NOTE THREE -->
          <div class="containerNoteTwo">
            <b>&#42;</b>3. Image PhotoShopped.<br>(the image is altered or fake)
          </div>
<!-- COPYRIGHT ACT -->
          <div class="copyrightAct">
            News quoted by OKthePK under the<br>provisions in Section 29 of the<br>Canadian Copyright Modernization Act.
          </div>
<!-- footer -->
          <div class="footer">
            <a href=["http://www.okthepk.ca/index.htm"](http://www.okthepk.ca/index.htm)><b class="footerPopup"><img class="noBorder"
            src="test04.gif" width="100" height="35" alt="" title=""><img class="footerPopupImg"
            src="test05.gif" alt="" title=""></b></a>
          </div>
        </div><!-- Responsive Text End -->
      </div>
    </div><br>
  </body>
</html>
```

**Comment by j9t** [comment]
@SLiMDataWrangler, when I copy your code above into https://validator.w3.org/#validate_by_input, the validator throws 11 errors.

Can you reproduce?

Where (URL) do you test?

**Comment by SLiMDataWrangler** [comment]
When uploading my code to the validator from the Microsoft Edge browser extension the validator gives no errors, only info about unnecessary slashes while my code contains NO slashes.

Perhaps the Microsoft Edge browser extension is causing the problem?

Where are the slashes coming from?


***@***.***

[signatureImage]

Canadian Railway News
www.okthepk.ca<http://www.okthepk.ca/>

Sooke News
www.okthepk.ca/sooke/news.htm<http://www.okthepk.ca/sooke/news.htm>

________________________________

**Comment by SLiMDataWrangler** [comment]
Here are 2 examples (3 screen captures) of a validation.

The first was by direct input of the code which resulted in 2 errors.

The second was by file upload from MS Edge browser extension which produces no errors and multiple Info about slashes which are not present in the original HTML code.

So were do the slashes come from?

And why no errors?

It looks like Microsoft is messing with the upload somehow?

![Image](https://github.com/user-attachments/assets/7d082495-f4cf-4beb-b77c-bd58b9bb19da)
![Image](https://github.com/user-attachments/assets/3d92e84f-3aae-4f06-b106-7a8aeae51e30)
![Image](https://github.com/user-attachments/assets/0a037388-a169-4524-bf61-1b6f401c1826)

**Comment by nschonni** [comment]
The first screenshot is showing the old validator at https://validator.w3.org/, which isn't this repo or project

**Comment by sideshowbarker** [comment]
> So were do the slashes come from?

No idea, but certainly not from us.

> And why no errors?

No idea, but whatever the answer to that might be, it's out of scope for our issue tracker here.

> It looks like Microsoft is messing with the upload somehow?

Maybe? But if so, there'd be nothing we could do here to help get that figured out and fixed.

**Comment by j9t** [comment]
Tangent:

> The first screenshot is showing the old validator at https://validator.w3.org/, which isn't this repo or project

@nschonni, yes, it does in the end validate agains the validator here (for living HTML). That we have two UIs (validator.w3.org, validator.nu) can be confusing yet there are also differences (e.g., the W3C validator would also validate previous HTML versions as well as XHTML).

(I haven’t been close to design decisions and mechanics so if someone else likes to correct or clarify, feel free!)
